### PR TITLE
Add ElPais.com translator

### PR DESCRIPTION
**Pull request by febrezo**
Added a new translator for an important Spanish newspaper. It improves default metadata extraction by adding authors, setting the item to newspaper, adding notes, snapshot, publication and others.

### CODE REVIEW

File: ElPais.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+function doWeb(doc, url) {
+	var resourceType = detectWeb(doc, url);
+
+	// Creating the item
+	var newItem = new Zotero.Item(resourceType);
+
+	//Setting common data:
+	newItem.url = url;
+	newItem.title = ZU.xpathText(doc, '//meta[@name="DC.title"]/@content');
```
Comment (adam3smith) [inline ElPais.js]: Zotero should already read most of these. Use the metadata translator to grab them: https://github.com/zuphilip/translators/wiki/Calling-metadata-translator

File: ElPais.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	newItem.date = ZU.xpathText(doc, '//meta[@name="DC.date"]/@content');
+	newItem.language = ZU.xpathText(doc, '//meta[@name="DC.language"]/@content');	
+	newItem.publication = ZU.xpathText(doc, '//meta[@name="DC.publisher"]/@content');
```
Comment (adam3smith) [inline ElPais.js]: That doesn't seem to get grabbed (at least in the text) but would this just be `El País`?

File: ElPais.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	newItem.place = ZU.xpathText(doc, '//span[@class="articulo-localizacion"]');
```
Comment (adam3smith) [inline ElPais.js]: Doesn't show in your test, but I'd assume this is the place from which this is reported, whereas place in Zotero is the publisher place. Not sure if that's even needed here, but if so it's probably just `Madrid`.

File: ElPais.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+	//Setting common data:
+	newItem.url = url;
+	newItem.title = ZU.xpathText(doc, '//meta[@name="DC.title"]/@content');
+	newItem.date = ZU.xpathText(doc, '//meta[@name="DC.date"]/@content');
+	newItem.language = ZU.xpathText(doc, '//meta[@name="DC.language"]/@content');	
+	newItem.publication = ZU.xpathText(doc, '//meta[@name="DC.publisher"]/@content');
+	newItem.place = ZU.xpathText(doc, '//span[@class="articulo-localizacion"]');	
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="DC.description"]/@content');	
+	newItem.tags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content').split(",");
```
Comment (adam3smith) [inline ElPais.js]: It'd be good to find an example with tags.

File: ElPais.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+	newItem.url = url;
+	newItem.title = ZU.xpathText(doc, '//meta[@name="DC.title"]/@content');
+	newItem.date = ZU.xpathText(doc, '//meta[@name="DC.date"]/@content');
+	newItem.language = ZU.xpathText(doc, '//meta[@name="DC.language"]/@content');	
+	newItem.publication = ZU.xpathText(doc, '//meta[@name="DC.publisher"]/@content');
+	newItem.place = ZU.xpathText(doc, '//span[@class="articulo-localizacion"]');	
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="DC.description"]/@content');	
+	newItem.tags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content').split(","); 	
+	// Getting the Author
+	newItem.creators = [Zotero.Utilities.cleanAuthor(ZU.xpathText(doc, '//meta[@name="author"]/@content'), "author", false)];
```
Comment (adam3smith) [inline ElPais.js]: Are we sure there are never multiple authors?

File: ElPais.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	
+	You should have received a copy of the Affero GNU General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) {
+	return "newspaperArticle";
```
Comment (adam3smith) [inline ElPais.js]: For newspapers we really would like to get search results imported as multiples

File: ElPais.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="DC.description"]/@content');	
+	newItem.tags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content').split(","); 	
+	// Getting the Author
+	newItem.creators = [Zotero.Utilities.cleanAuthor(ZU.xpathText(doc, '//meta[@name="author"]/@content'), "author", false)];	
+	
+	// Adding the attachment
+	newItem.attachments.push({
+		title: "ElPaís.com Snapshot",
+		mimeType: "text/html",
+		url: url
```
Comment (adam3smith) [inline ElPais.js]: use `document: doc`, which grabs the current document instead of reloading it from the URL.

**Comment by zuphilip** [comment]
Superseeded by #1514

**Pull request closed without merging**